### PR TITLE
pfschecker OK

### PR DESCRIPTION
**Pull request by EllipticEllipsis**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

Also did some cleanup of contpfs.

... so what is a pfs anyway?

### CODE REVIEW

File: src/libultra/io/pfschecker.c [hunk 1]
```diff
@@ -1,7 +1,213 @@
...
+        }
+        for (; j < PFS_INODE_SIZE_PER_PAGE; j++) {
+            checkedInode.inodePage[j].ipage = PFS_PAGE_NOT_USED;
+        }
+
+        for (j = 0; j < pfs->dir_size; j++) {
+            while (nextNodeInFile[j].inode_t.bank == bank &&
+                   nextNodeInFile[j].ipage >= (u16)pfs->inodeStartPage) { // cast required
+                u8 val;
+                val = nextNodeInFile[j].inode_t.page;
```
Comment (Thar0) [inline src/libultra/io/pfschecker.c:126]:
```suggestion
                u8 val = nextNodeInFile[j].inode_t.page;

```

**Review by Thar0 — CHANGES_REQUESTED** [review]
"Pak File System"

**Comment by EllipticEllipsis** [comment]
As in controller pak or game pak? Or do both use the same?

**Comment by Thar0** [comment]
Controller pak. The "game pak" is the cartridge, which has no inherent filesystem. `z_std_dma.c` and `dmadata` implement a kind of filesystem for that.

**Review by Thar0 — APPROVED** [review]

**Review by kyleburnette — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**